src/DocumentFormat.OpenXml/Framework/Schema/ParticleExtensions.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using DocumentFormat.OpenXml.Framework.Schema;
+using DocumentFormat.OpenXml.Validation.Schema;
+
+namespace DocumentFormat.OpenXml.Framework
+{
+    internal static class ParticleExtensions
+    {
+        public static ParticleCollection<TElement> GetCollection<TElement>(this CompiledParticle compiled, OpenXmlCompositeElement element)
+            where TElement : OpenXmlElement
+            => new ParticleCollection<TElement>(compiled, element);
+
+        public static TElement Get<TElement>(this CompiledParticle compiled, OpenXmlCompositeElement element)
```

Comment:
I agree and thought about doing that way, but this makes it a lot easier to test without needing to set up a whole composite element. I'll add a comment to that effect (it's internal, so it can always be changed)